h/services/bulk_api/annotation.py

```diff
@@ -74,6 +85,11 @@ def _search_query(cls, authority, username, created) -> Select:
             .select_from(AnnotationSlim)
             .join(cls._AUTHOR, cls._AUTHOR.id == AnnotationSlim.user_id)
             .join(Group, Group.id == AnnotationSlim.group_id)
+            .join(cls._ANNOTATION)
+            .outerjoin(
+                cls._ANNOTATION_PARENT,
+                cls._ANNOTATION.references[0] == cls._ANNOTATION_PARENT.id,
```

Comment:
I'm guessing `references[0]` is the root annotation of the thread? What if a reply in the middle of a thread is hidden, and there are replies to that hidden reply?